### ISSUE TITLE
Accessibility: Link text used for multiple different destinations - Visit Exhibit button

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#416]
**Description**
The text "Visit Exhibit" is used multiple times but all go to different locations. Accessibility expects that identical text should always go to the same location.

**Solution**
Option 1: Make each link text unique per exhibit.
Option 2: Add an "aria-labelledby" element and have it point to the exhibit title on the back of the card.

**Opinion**
I would recommend going with Option 2 for being able to keep the button text shorter and have it still make sense for all users.

**Comment by elrayle** [comment]
Addressed by PR https://github.com/projectblacklight/spotlight/pull/2659

Closing of this issue is pending:

* merge in Spotlight release-2.x
* new release of Spotlight 2.x series
* update of our Spotlight to the new 2.x release

**Comment by chrisrlc** [comment]
No longer an issue in spotlight 3.x
